pkg/provisioner/api/provisioner.go

```diff
@@ -43,6 +43,8 @@ type Provisioner interface {
 	Provision(options *BucketOptions) (*v1alpha1.ObjectBucket, error)
 	// Grant should be implemented to handle access to existing buckets
 	Grant(options *BucketOptions) (*v1alpha1.ObjectBucket, error)
+	// Update should be implemented to handle updation of existing buckets
+	Update(options *BucketOptions) error
```

Comment:
I think this is a breaking change in that **all** provisioners will need to add an Update method when they recompile.

Reply:
That happens only if they update the latest version of the library right? Can we release note or something in which we can mention about this change

Reply:
Agree